docs/guides/chunks-storage-getting-started.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: "Getting Started with Chunks Storage"
```

Comment:
Because of this change, could you fix the link in `docs/case-studies/buoyant-cloud.md` please?

Reply:
Also, what if we do a further step and move this doc under `docs/chunks-storage/`?

Reply:
I think that makes sense. @pracucci WDYT about also moving the `docs/guides/running-chunks-storage-in-production.md` and ` docs/guides/running-chunks-storage-with-cassandra.md` under that section as well?

Reply:
I think it makes sense. We're heading towards deprecating the chunks storage (even if very soon) so I agree having the blocks storage doc to be more prominent.